app/main/views.py

```diff
@@ -8,6 +9,15 @@ def index():
     return render_template('main/index.html')
 
 
+@main.route('/map')
```

Comment:
You should combine this with the index ('/') route. 

@kongsally we don't need the current index route do we?